go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.1.2
 	github.com/hashicorp/go-azure-helpers v0.47.0
-	github.com/hashicorp/go-azure-sdk v0.20230127.1133443
+	github.com/hashicorp/go-azure-sdk v0.20230130.1140358
```

Comment:
could we update this to the latest in a separate PR?

Reply:
https://github.com/hashicorp/terraform-provider-azurerm/pull/20239